### PR TITLE
Removed vote weight section from README

### PR DESCRIPTION
**Pull request by golergka**
Since [relevant pull request](https://github.com/chaosbot/chaos/pull/42) have been merged, I feel we need to update documentation.

### CODE REVIEW

**Comment by Nonepse** [comment]
Shouldn't the bit about the threshold be kept?

**Comment by golergka** [comment]
@Nonepse you're absolutely right. Will edit

**Comment by namibj** [comment]
+1

**Comment by sanjaybv** [comment]
I agree with @Nonepse and it doesn't say anything about minimum voting requirement.

**Comment by Nonepse** [comment]
Speaking about it, maybe that could be more detailed:
> a percentage of the number of watchers on the repository

What is the percentage?

**Comment by reddraggone9** [comment]
👎 There's already a [PR](https://github.com/chaosbot/chaos/pull/60) open to document #42 and this PR still hasn't been updated to leave in the bit about thresholds.

**Comment by Ajedi32** [comment]
This is a duplicate of #60.

**Comment by reddraggone9** [comment]
@Nonepse 
> What is the percentage?

**TL;DR**: It looks like one vote over 50% of the *votes* must be up votes right now.

[All accounts which are old enough have their votes count for 1 point.][vote weight] All of the votes are then [summed up][vote sum] (up votes are positive and down votes negative). The PR is [approved if the sum is at or above the specified threshold][vote approval]. That [approval threshold] is currently 1 (i.e. 1 vote), but it can be easily modified to instead be a [fraction of the number of watchers of this repo][watchers scale factor].

[vote weight]: https://github.com/chaosbot/chaos/blob/fda0309a097a59d57474133e79fa314171073538/github_api/voting.py#L111-L124
[vote sum]: https://github.com/chaosbot/chaos/blob/fda0309a097a59d57474133e79fa314171073538/github_api/voting.py#L127-L135
[vote approval]: https://github.com/chaosbot/chaos/blob/fda0309a097a59d57474133e79fa314171073538/chaos.py#L92-L95
[approval threshold]: https://github.com/chaosbot/chaos/blob/fda0309a097a59d57474133e79fa314171073538/github_api/voting.py#L138-L143
[watchers scale factor]: https://github.com/chaosbot/chaos/blob/fda0309a097a59d57474133e79fa314171073538/settings.py#L43-L47

**Comment by MCSH** [comment]
Duplicate of #60
👎

**Comment by MCSH** [comment]
How did it got accepted with 2 negative votes..?

**Comment by Ajedi32** [comment]
@MCSH It doesn't need to be unanimous. The vote total was 28 for, 4 against. Majority wins.

**Comment by MCSH** [comment]
@ajedi32
Interesting. How can i see anonymous votes and how can I cast one?

Nvm i realized you cant see reactions on phone. And thats why i didnt saw it
